allure-report-face/package.json

```diff
@@ -45,6 +45,7 @@
     "proxyquire": "^1.7.3",
     "style-loader": "^0.12.4",
     "underscore.string": "^3.2.2",
+    "url-loader": "^0.5.7",
```

Comment:
elliptics was not served csv correctly, but Vanya deployed a fix yesterday. But I've not change this since that time